src/cmake/modules/CUnit/src/main.c.in

```diff
@@ -185,15 +185,15 @@ int main(int argc, char *argv[])
 
     if ((err = parse_options(argc, argv)) != CUE_SUCCESS) {
         usage(argv[0]);
-        return err;
+        return (int)err;
     } else if (opts.print_help) {
         help(argv[0]);
         return CUE_SUCCESS;
```

Comment:
The change to blindly casting a ``CU_ErrorCode`` to an``int`` only makes a small oversight worse. Exit code on success should simply be 0 [*], exit code on error should reflect (in some form or manner) the kind of problem encountered. Here, just signalling failure would be good enough as far as I'm concerned, but arguably a failure to initialise the registry is of a different kind than providing invalid options.

[*] ``EXIT_SUCCESS`` if you're pedantic ... but that gets into trouble if you want to be able to return different exit codes for different kinds of problems, because the only proper alternative to ``EXIT_SUCCESS`` is ``EXIT_FAILURE``. So please just 0 for success and 1 .. 126 or so for errors.